cumulus/test/service/src/lib.rs

```diff
@@ -400,7 +400,8 @@ pub fn node_config(
 		offchain_worker: OffchainWorkerConfig { enabled: true, indexing_enabled: false },
 		force_authoring: false,
 		disable_grandpa: false,
-		dev_key_seed: Some(key_seed),
+		// Force using Alice as the dev seed as the specified executor authority in primary chain spec is Alice.
+		dev_key_seed: Some(Keyring::Alice.to_seed()),
```

Comment:
This doesn't seem right. You'll have to change it back in order to support multiple identities, why not specifying correct `key_seed` from the outside instead?

Reply:
I was hesitant/lazy to fix the node name and dev key seed properly :P, it works for now, but you're right, we'll have to fix it in the future anyway, updated in https://github.com/subspace/subspace/pull/523/commits/1057adc3131b433c9e63edb9a623d4751a11fbe1.

Also rebased on the latest main, only a few minor conflicts.